nixos/modules/services/web-apps/mastodon.nix

```
@@ -100,6 +100,36 @@ let
     eval -- "\$@"
   '';
 
+  sidekiqUnits = lib.attrsets.mapAttrs' (name: processCfg: 
```

```suggestion
  sidekiqUnits = lib.attrsets.mapAttrs' (name: processCfg:
```

Remove trailing whitespace to fix failing `editorconfig` check: https://github.com/NixOS/nixpkgs/actions/runs/3526803582/jobs/5915148177

---

thanks!